content/en/docs/Installation/customization.md

```diff
@@ -259,6 +259,62 @@ The following flags can be used to control the advanced behavior of the various
+36. `profile` (ABCR): setting this flag to `true` will enable profiling. Default is `false`.
+37. `profileAddress` (ABCR): Configures the address of the profiling server. Default is `""`.
+38. `profilePort` (ABCR): specifies port to enable profiling. Default is `6060`.
+39. `protectManagedResources` (A): protects the Kyverno resources from being altered by anyone other than the Kyverno Service Account. Defaults to `false`. Set to `true` to enable.
+40. `registryCredentialHelpers` (ABR): enables cloud-registry-specific authentication helpers. Defaults to `"default,google,amazon,azure,github"`.
+41. `reportsChunkSize` (R): maximum number of results in generated reports before splitting occurs if there are more results to be stored. Default is `1000`.
+42. `serverIP` (AC): like the `kubeconfig` flag, used when running Kyverno outside of the cluster which it serves.
+43. `servicePort` (AC): port used by the Kyverno Service resource and for webhook configurations. Default is `443`.
+44. `skipResourceFilters` (R): obey the ConfigMap's resourceFilters when performing background report scans. Default is `true`.
+45. `skip_headers` (ABCR): if true, avoid header prefixes in the log messages.
+46. `skip_log_headers` (ABCR): if true, avoid headers when opening log files (no effect when -logtostderr=true).
+47. `stderrthreshold` (ABCR): logs at or above this threshold go to stderr when writing to files and stderr (no effect when -logtostderr=true or -alsologtostderr=false). Default is `2`.
+48. `tracingAddress` (ABCR): tracing receiver address, defaults to `''`.
+49. `tracingCreds` (ABCR): set to the CA secret containing the certificate which is used by the Opentelemetry Tracing Client. If empty string is set, an insecure connection will be used.
+50. `tracingPort` (ABCR): tracing receiver port. Default is `"4317"`.
+51. `transportCreds` (ABCR): set to the CA secret containing the certificate used by the OpenTelemetry metrics client. Empty string means an insecure connection will be used. Default is `""`.
+52. `v` (ABCR): sets the verbosity level of Kyverno log output. Takes an integer from 1 to 6 with 6 being the most verbose. Level 4 shows variable substitution messages. Default is `2`.
+53. `vmodule` (ABCR): comma-separated list of pattern=N settings for file-filtered logging.
+54. `webhookRegistrationTimeout` (A): specifies the length of time Kyverno will try to register webhooks with the API server. Defaults to `120s`.
+55. `webhookTimeout` (A): specifies the timeout for webhooks. After the timeout passes, the webhook call will be ignored or the API call will fail based on the failure policy. The timeout value must be between 1 and 30 seconds. Defaults is `10s`.
+56. `ttlReconciliationInterval` (C): specifies the interval after which the resource controller reconciliation should occur for the ttl-controller-manager.
+
```

Comment:
Why all these changes ?

Reply:
Because any time you add something to a numerical list all the subsequent values are updated.